docs/data/material/varnish/logos/ProjectsLogosDemo.js

```diff
@@ -72,6 +72,48 @@ export default function Demo() {
           <logos.Incubator />{' '}
         </Center>
       </Item>
+
+      <Item>
+        <Center>
+          <ImageListItemBar title="EarthRanger" position="top" />
+          <logos.EarthRanger width="32" height="32" />{' '}
```

Comment:
```suggestion
          <logos.EarthRanger width="32" height="32" />
```
see others above and below also